content/en/xtc/privacy/_index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Privacy"
+
+type: docs
+
+weight: 900
+
+description: How XTC processes your personal data.
+---
+
+You can find all the details about the processing of personal data on Xceptance websites in the [Xceptance Privacy Statement](https://www.xceptance.com/en/contact/privacy.html). This is the condensed version of how XTC is handling personal data:
+
+### Cookie Usage
+We use only necessary cookies to make our site work. Necessary cookies enable core functionality such as security, network management, and accessibility. You cannot disable these without breaking the functionality.
+
+### Data Storage (Hosting)
+All resources are hosted internally at _xtc.xceptance.com_. 
+
+{{< TODO / >}}Where is data hosted physically? By which hosting provider (Hetzner)?
+
+{{< TODO / >}}Which data is logged by hosting provider and how long is it kept?
+
+### Data Privacy (Analytics/Tracking)
+XTC includes no external analytics services or tracking.
+
+None of your data generated by or entered in XTC will be shared with third parties.
+
+### Data Processing (Account and Usage Data)
+In order to use XTC, you need to create an account. The information and personal data you share with us will be used solely for the use of XTC services. This information is: 
+* **Required:**
+    * First Name
+    * Last Name
+    * Email
+* **Optional:**
+    * Phone Number (for monitoring notifications via text message)
+* {{< TODO / >}}usage data, which is probably stored for audit log etc.? 
+
+XTC does _not_ store any of your contractual information or payment data.
+
+### Data Deletion
+XTC does not do automated account management (i.e. deactivation of accounts).
```

Comment:
Accounts don't expire currently, only open and unaccepted invocations will (soon) expire.